### ISSUE TITLE
Data sampling

### ISSUE THREAD

**Issue by hbellafkir** [technicolor-research/subic#4]
hi,

I just read the paper and found out that e.g.  for Cifar-10 only validation images are used as db and queries which not comon and most the other papers  (hashnet, dph, dqn, dtq,... ), where they use the training images also as db and validation images as queries. any reason for that? 

thanks in advance

**Comment by himalayajain** [comment]
Hi,

Sorry for the long delay in noticing your message.
The task is semantic retrieval thus, using the training images as the database is a bit futile given the **annotated training data.** This is because one can directly use the labels of training images to index/encode them.
